src/BjyAuthorize/Service/Authorize.php

```diff
@@ -348,12 +349,21 @@ protected function loadResource($resources, $parent = null)
      */
     protected function loadRule(array $rule, $type)
     {
-        $privileges = $assertion = null;
+        $privileges = $assertions = null;
         $ruleSize   = count($rule);
 
         if (4 === $ruleSize) {
-            list($roles, $resources, $privileges, $assertion) = $rule;
-            $assertion = $this->serviceLocator->get($assertion);
+            list($roles, $resources, $privileges, $assertions) = $rule;
+            if (is_array($assertions) && count($assertions) > 1) {
```

Comment:
Well, I've several assertions that can be combined. I don't really want to create a service for each combination.